tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+    "\n",
+    "`% gmx solvate -cp boxed.gro -cs -o solvated.gro -p topol.top`\n",
+    "\n",
+    "You may have noticed in some of the output generated that the total system charge is +4. In order for us to use an Ewald\n",
+    "method to calculate the electrostatic interactions we need to have a neutral system overall. Therefore we will add\n",
+    "counterions (chloride ions, in this case) using the option -neutral and enough ions to make the solution up to 150 mM (-conc 0.15). This is done by replacing random water molecules (SOL) with NA+ or CL- ions.\n",
+    "\n",
+    "`% gmx grompp -c solvated.gro -p topol.top -f /home/shared/data/genion.mdp -o genion.tpr`\\\n",
+    "`% gmx genion -s genion.tpr -conc 0.15 -neutral -pname NA -nname CL -o system.gro -p topol.top`\n",
+    "\n",
+    "When prompted, enter the group that corresponds to SOL (should be 13 or thereabouts).\n",
+    "\n",
+    "### Energy Minimization\n",
+    "\n",
+    "Before we can run the actual dynamics, we need to first minimize the energy of the system. Ideally you would minimize down until the forces were below a certain level (tolerance), but we will just give a quick burst of 200 steps here. Since we have finished setting up the system, we will now move to a new directory to perform our simulation from:\n",
+    "\n",
+    "`% cd ../`\n",
+    "\n",
+    "`% mkdir run`\n",
+    "\n",
+    "`% cd run`\n",
+    "\n",
+    "The `grompp` command will read the information of the system that we will provide (coordinates, topologies and simulation parameters) and will generate a run input file:\n",
+    "\n",
+    "`% gmx grompp -c ../setup/system.gro -p ../setup/topol.top -f /home/shared/data/em.mdp -o em.tpr`\n",
+    "\n",
+    "We cannot run the simulation on the login nodes of ARC; these can only be used to prepare the system for the simulation. Therefore instead of typing the `mdrun` command -that will initiate the energy minimisation- directly on the command line, we will submit a script that will submit the job to the job scheduler. Copy this script to your working directory:\n",
+    "\n",
+    "`% cp /home/shared/data/submit_em.sh .`\n",
+    "\n",
+    "You can explore its contents and see that the last line in the file contains the `mdrun` command by typing: \n",
+    "\n",
+    "`% cat submit_em.sh`\n",
+    "\n",
+    "The file contents will be printed on your terminal.\\\n",
+    "Now submit it to the cluster queue.\n",
+    "\n",
+    "`% sbatch submit_em.sh`\n",
+    "\n",
+    "It will take a few minutes to run, depending on the waiting times of the queue. Check on the status of the run by typing:\n",
+    "\n",
+    "`% squeue -u username`\n",
+    "\n",
+    "Remember to replace `username` with your own username! It should print something like this:\n",
+    "\n",
+    "`             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON) \n",
```

Comment:
The alignment here isn't great on my end, are you getting the correct alignment?

Reply:
Tried to fix that, I think it is better now.